system-tests/services/_scripts/external-volumes/teardown

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+pip install boto3
+pip install retrying
+
+cmd="./services/_scripts/external-volumes/volclean.py \
+-y -r $REGION -k $ACCESS_KEY_ID -s \"$SECRET_ACCESS_KEY\" \
+--ignore-metrics --tags 'Name:^integration-test-dcos-ui-$TEST_UUID'"
+
+MATCH_STRING="*Removing*Volumes*"
+TIMEOUT=30
+counter=0
+
+num_vols_delete=$NUM_VOLS_DELETE
```

Comment:
The purpose of NUM_VOLS_DELETE is to iteratively delete each volume. The volumes must be detached before the script can find the volume. It is done this way instead of having an arbitrary delay to wait for the volumes to detach.